subprojects/logging/src/main/java/org/gradle/internal/logging/sink/GroupingProgressLogEventGenerator.java

```diff
@@ -100,7 +100,7 @@ private void onStart(ProgressStartEvent startEvent) {
     }
 
     private boolean isGroupedOperation(BuildOperationCategory buildOperationCategory) {
-        return buildOperationCategory == BuildOperationCategory.TASK || buildOperationCategory == BuildOperationCategory.CONFIGURE_PROJECT;
+        return buildOperationCategory == BuildOperationCategory.TASK || buildOperationCategory == BuildOperationCategory.CONFIGURE_PROJECT || buildOperationCategory == BuildOperationCategory.TRANSFORM;
```

Comment:
We could do this with an {{EnumSet.contains()}} instead of a method. In any case, please split this to multiple lines.